DigitalLearningSolutions.Web.Tests/Controllers/TrackingSystem/CentreConfiguration/RegistrationPromptsControllerTests.cs

```diff
@@ -276,6 +276,39 @@ public void AddRegistrationPromptConfigureAnswers_returns_error_with_unexpected_
             result.Should().BeRedirectToActionResult().WithControllerName("LearningSolutions").WithActionName("Error");
         }
 
+        [Test]

```

Comment:
I'd probably add a quick check that the TempData is cleared into this test.

Maybe also worth adding a quick test for the error page redirect that we've added?